Marketstack/Interfaces/IMarketstackService.cs

```diff
@@ -11,6 +11,6 @@ public interface IMarketstackService
         Task<List<Exchange>> GetExchanges();
         Task<List<Stock>> GetExchangeStocks(string exchangeMic);
         Task<List<StockBar>> GetStockEodBars(string stockSymbol, DateTime fromDate, DateTime toDate);
-        Task<List<StockBar>> GetStockIntraDayBars(string stockSymbol, DateTime fromDate, DateTime toDate);
+        Task<List<StockBar>> GetStockIntraDayBars(string stockSymbol, DateTime fromDate, DateTime toDate, string interval = "1hour");
```

Comment:
Please move the `interval` to an Enum.

Reply:
Problem is that enums does not allow numbers as a part of their key.

Reply:
Hm, found a way using description attributes

Reply:
Ok?